### PR TITLE
add logging of 500 errors

### PR DESCRIPTION
**Pull request by LasseRaatz**
that's a lot of errors xd

### CODE REVIEW

File: backend/src/middleware/error.js [hunk 1]
```diff
@@ -8,11 +9,12 @@ module.exports = (err, _req, res, next) => {
   if (err.message === 'Unauthorized') {
     return res.status(403).send(new Error('You are not authorized to use this resource'));
   }
-
+
```
Comment (DagBAndersen) [inline backend/src/middleware/error.js]:
```suggestion

```

Comment (LasseRaatz) [reply backend/src/middleware/error.js]: I don't concern myself with such banalities

File: backend/src/middleware/error.js [hunk 2]
```diff
@@ -8,11 +9,11 @@ module.exports = (err, _req, res, next) => {
   if (err.message === 'Unauthorized') {
     return res.status(403).send(new Error('You are not authorized to use this resource'));
   }
-
   if (err.message.toLowerCase().includes('not found')) {
     return res.status(404).send(new Error(err.message));
   }
```
Comment (DagBAndersen) [inline backend/src/middleware/error.js]:
```suggestion

  if (err.message.toLowerCase().includes('not found')) {
    return res.status(404).send(new Error(err.message));
  }
```

Comment (LasseRaatz) [reply backend/src/middleware/error.js]: my father is capable of beating up your father

**Pull request merged**